scenarios/MTA/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.4
+
+LABEL vendor="Red Hat inc."
+LABEL maintainer="CSPI QE Team"
+USER root
+
+RUN set -x && \
```

Comment:
Even though `yum` is available on UBI8, I would suggest using `dnf` as its the default package manager moving forward.